lib/readline.js

```diff
@@ -868,6 +864,14 @@ Interface.prototype._ttyWrite = function(s, key) {
         break;
 
       default:
+        if (key.name === 'tab') {
+          // If tab completion enabled, do that...
+          if (typeof this.completer === 'function') {
+            this._tabComplete();
+            break;
+          }
+        }
```

Comment:
I believe that's how I initially did it, but then `make jslint` complained that the `case` did not end with a `break`. So I changed it to the arrangement in the PR.